### PR TITLE
Test the Civic Cookie Control banner

### PR DESCRIPTION
**Pull request by kenoir**
## Who is this for?

People who want to comply with data protection legislation around cookies!

This work builds on https://github.com/wellcomecollection/wellcomecollection.org/pull/10638

Part of https://github.com/orgs/wellcomecollection/projects/18/views/1?pane=issue&itemId=53236002

## What is it doing for them?

Trying out a 3rd party library to see if it behaves as expected.

### CODE REVIEW

File: common/views/pages/_document.tsx [hunk 1]
```diff
@@ -16,14 +16,63 @@ import {
...
+                      description: 'Analytical cookies help us to improve our website by collecting and reporting information on its usage.',
+                      cookies: ['_ga', '_ga*', '_gid', '_gat', '__utma', '__utmt', '__utmb', '__utmc', '__utmz', '__utmv'],
+                      onAccept: function(){
+                          console.log('analytics accepted');
+                      },
+                      onRevoke: function(){
+                          console.log('analytics rejected');
+                          // TODO: Hoping we can listen for this in a component and react to it!
+                          const event = new CustomEvent('analyticsConsentWithdrawn', {});
+                          window.dispatchEvent(event);
```
Comment (kenoir) [inline common/views/pages/_document.tsx:65]: I can add a listener via the console and see this event being fired, so this bit works.

File: common/views/pages/_document.tsx [hunk 2]
```diff
@@ -16,14 +16,63 @@ import {
...
-import { getConsentCookieServerSide } from '@weco/common/utils/cookie-consent';
+import { getConsentState } from '@weco/common/utils/cookie-consent';
 
 const {
   ANALYTICS_WRITE_KEY = '78Czn5jNSaMSVrBq2J9K4yJjWxh6fyRI',
   NODE_ENV = 'development',
 } = process.env;
 
+const CookieControl: FunctionComponent = () => {
+  // TODO: I'd expect this to shout when the event is fired, but it doesn't :\
```
Comment (kenoir) [inline common/views/pages/_document.tsx:27]: See https://github.com/wellcomecollection/wellcomecollection.org/pull/10654/files#diff-d32d179af5b0584007ab20b27d535dc56eeb4618235b77e7e15463a156329946R63

**Comment by rcantin-w** [comment]
@kenoir would we be ok to close?

**Comment by kenoir** [comment]
Closing in favour of https://github.com/wellcomecollection/wellcomecollection.org/pull/10663

**Pull request closed without merging**